frame-metadata/src/v13.rs

```diff
@@ -66,42 +73,57 @@ pub struct OuterEventMetadata {
 #[derive(Clone, PartialEq, Eq, Encode, Debug)]
 #[cfg_attr(feature = "std", derive(Decode, Serialize))]
 pub struct EventMetadata {
+	/// Name of the event.
 	pub name: DecodeDifferentStr,
+	/// Arguments of the event.
 	pub arguments: DecodeDifferentArray<&'static str, StringBuf>,
+	/// Documentation of the event.
 	pub documentation: DecodeDifferentArray<&'static str, StringBuf>,
 }
 
 /// Metadata about one storage entry.
 #[derive(Clone, PartialEq, Eq, Encode, Debug)]
 #[cfg_attr(feature = "std", derive(Decode, Serialize))]
 pub struct StorageEntryMetadata {
+	/// Variable name of the storage entry.
 	pub name: DecodeDifferentStr,
+	/// A storage modifier of the storage entry (is it optional? does it have a default value?).
```

Comment:
Specifically this refers to the "return type" when fetching a storage entry, and what the value will be if the key is not present. 

- `Optional` means you should expect an `Option<MyValue>`, with `None` returned if the key is not present, 
- `Default` means you should expect a `MyValue` with the default value of `default` if the key is not present.

Reply:
tyvm